plugins/decisions.rb

```diff
@@ -25,7 +25,7 @@ def match?(cmd)
   match /decide (.+)/, method: :decide, strip_colors: true
 
   def decide(m, query)
-    choices = query.split(/ (?:OR|or|\|\|) /).map(&:strip).map(&:downcase).uniq
+    choices = query.split(/ (?:OR|or|\|\|) |,/).map(&:strip).map(&:downcase).uniq
```

Comment:
Why is `|,` outside of the main regexp group?

Reply:
the main regex group matches it with spaces, so ` , `. I needed `,` `, ` and ` ,` to match as well

Reply:
Okay, that makes sense. Thanks for the explanation.